### ISSUE TITLE
D.2 Variant Types

### ISSUE THREAD

**Issue by heftberger** [FIAF/fiaf-cataloguing-manual#86]
We should look into this list and discuss if we should keep it or rather provide examples.

**Comment by heftberger** [comment]
Move D.2 to Variant Chapter.

**Comment by heftberger** [comment]
Do we need a new branch in order to do that?

**Comment by natashafairbairn** [comment]
I don't think so. I have already moved D.2. Variant types into the new Variants chapter in the branch created for Issue 131 "new texts for variant chapter".
